### DISCUSSION TITLE
keep getting dead kernel

### DISCUSSION THREAD

**Discussion by dentadelta** [lucidrains/imagen-pytorch#189] [Q&A]
Keep getting dead kernel when running the below code block

for i in (1, 2):
        loss = imagen(pixel_values, texts = captions, unet_number = i)

**Comment by dentadelta** [comment]
I think it's because I'm not using 256 x 256 image size so it doesnt match with the built-in T5 text embedded size

**Reply by wasyleque** [reply]
You should include debug (errors) to Your post I guess ...
